plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/AbstractTrinoCatalog.java

```diff
@@ -264,6 +283,35 @@ protected void deleteTableDirectory(TrinoFileSystem fileSystem, SchemaTableName
         }
     }
 
+    protected String createMaterializedViewStorage(ConnectorSession session, SchemaTableName viewName, ConnectorMaterializedViewDefinition definition)
+    {
+        String storageTableName = IcebergTableName.tableNameWithType(viewName.getTableName(), MATERIALIZED_VIEW_STORAGE);
+        Map<String, Object> storageTableProperties = new HashMap<>(definition.getProperties());
+        storageTableProperties.putIfAbsent(FILE_FORMAT_PROPERTY, DEFAULT_FILE_FORMAT_DEFAULT);
+
+        String storageSchema = getStorageSchema(definition.getProperties()).orElse(viewName.getSchemaName());
```

Comment:
Let's skip the call to `getStorageSchema`. It would be good to deprecate the STORAGE_SCHEMA mv property in favor of the new approach.

The STORAGE_SCHEMA was introduced basically to hide those MV storage tables, which we're doing now better.